src/InstallationDirectories.cc

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2023 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gz/rendering/config.hh>
+#include <gz/rendering/InstallationDirectories.hh>
+
+#include <gz/common/Filesystem.hh>
+
+namespace gz
+{
+namespace rendering
+{
+inline namespace GZ_RENDERING_VERSION_NAMESPACE {
+
+std::string getResourcePath()
+{
+  return gz::common::joinPaths(getInstallPrefix(), GZ_RENDERING_RELATIVE_RESOURCE_PATH);
+}
+
+std::string getEngineInstallDir()
+{
+  return gz::common::joinPaths(getInstallPrefix(), GZ_RENDERING_ENGINE_RELATIVE_INSTALL_DIR);
```

Comment:
CI picked up a couple of minor issues here:

```
  /github/workspace/src/InstallationDirectories.cc:31:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
  /github/workspace/src/InstallationDirectories.cc:36:  Lines should be <= 80 characters long  [whitespace/line_length] [2]
```

Reply:
Thanks! I did not notice that as I tought the CI errors were all due to the gz-cmake 3.1.0 missing on Focal.

Reply:
Fixed in https://github.com/gazebosim/gz-rendering/pull/804/commits/c4129159e3715e10b31b35342d589eb01a1be38d .